crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -481,7 +614,7 @@ impl SyncService {
                 // No need to stop if we were not running.
                 return Ok(());
             }
-            State::Running => (),
+            State::Running | State::Offline => (),
```

Comment:
preexisting/nit: let's avoid using `()` and instead use `{}` to materialize an empty statement

Reply:
5df0f2a3f3f5d3b78b65de6f738a03582ca11a31